openpype/settings/defaults/project_anatomy/templates.json

```diff
@@ -58,12 +58,16 @@
             "file": "{originalBasename}.{ext}",
             "path": "{@folder}/{@file}"
         },
+        "transient": {
+            "folder": "{root[work]}/{project[name]}/{hierarchy}/{asset}/work/{family}/{subset}"
+        },
```

Comment:
Is it weird that we're adding this default but not have default custom staging dir profiles defined using this? I can see in the code that it defaults to `transient` if the template name setting is kept empty - but isn't that maybe a bit too implicit?

Maybe we should focus on creating good documentation for the feature as opposed to needing a default template key defined but enforce an explicit template value to be set with it.